torch/fft/__init__.py

```diff
@@ -203,14 +218,17 @@
 Computes the N dimensional discrete Fourier transform of :attr:`input`.
 
 Note:
-
     The Fourier domain representation of any real signal satisfies the
     Hermitian property: ``X[i_1, ..., i_n] = conj(X[-i_1, ..., -i_n])``. This
     function always returns all positive and negative frequency terms even
     though, for real inputs, half of these values are redundant.
     :func:`~torch.fft.rfftn` returns the more compact one-sided representation
     where only the positive frequencies of the last dimension are returned.
 
+Note:
+    Supports torch.half and torch.chalf on CUDA with GPU Architecture SM53 or greater.
+    However it only supports powers of 2 signal size along the dimensions of FFT.
```

Comment:
"dimensions of the FFT" is a bit unclear, can we say "in every transformed dimension". For hermitian signals, also be sure to say that `2(n-1)` must be a power to two in the symmetric dimension.